charts/argocd-apps/values.yaml

```diff
@@ -97,6 +97,10 @@ applicationsets: []
 #   namespace: argocd
 #   additionalLabels: {}
 #   additionalAnnotations: {}
+#   # Ref https://argo-cd.readthedocs.io/en/stable/operator-manual/applicationset/Controlling-Resource-Modification/#allow-temporarily-toggling-auto-sync
+#   ignoreAppDifferences:
+#   - jsonPointers:
+#     - /spec/syncPolicy
```

Comment:
Can you please add the attributes in CI as well ? 
https://github.com/argoproj/argo-helm/blob/main/charts/argocd-apps/ci/applicationsets-values.yaml